modules/core/src/test/scala/me/mnedokushev/zio/apache/parquet/core/codec/SchemaEncoderDeriverSpec.scala

```diff
@@ -25,6 +25,53 @@ object SchemaEncoderDeriverSpec extends ZIOSpecDefault {
+    g:Int, h:Int, i:Int, j:Int, k:Int, l:Int, m:Int, n:Int,
+    o:Int, p:Int, q:Int, r:Int, s:Int, t:Int, u:Int, v:Int,
+    w:Int
+  )
+  object BigRecord {
+    implicit val schema: Schema[BigRecord] = DeriveSchema.gen[BigRecord]
+  }
+
+  case class MaxArityRecord(
+    f1:Int, f2:Int, f3:Int, f4:Int, f5:Int, f6:Int, f7:Int, f8:Int, f9:Int, f10:Int,
+    f11:Int, f12:Int, f13:Int, f14:Int, f15:Int, f16:Int, f17:Int, f18:Int, f19:Int, f20:Int,
+    f21:Int, f22:Int, f23:Int, f24:Int, f25:Int, f26:Int, f27:Int, f28:Int, f29:Int, f30:Int,
+    f31:Int, f32:Int, f33:Int, f34:Int, f35:Int, f36:Int, f37:Int, f38:Int, f39:Int, f40:Int,
+    f41:Int, f42:Int, f43:Int, f44:Int, f45:Int, f46:Int, f47:Int, f48:Int, f49:Int, f50:Int,
+    f51:Int, f52:Int, f53:Int, f54:Int, f55:Int, f56:Int, f57:Int, f58:Int, f59:Int, f60:Int,
+    f61:Int, f62:Int, f63:Int, f64:Int, f65:Int, f66:Int, f67:Int, f68:Int, f69:Int, f70:Int,
+    f71:Int, f72:Int, f73:Int, f74:Int, f75:Int, f76:Int, f77:Int, f78:Int, f79:Int, f80:Int,
+    f81:Int, f82:Int, f83:Int, f84:Int, f85:Int, f86:Int, f87:Int, f88:Int, f89:Int, f90:Int,
+    f91:Int, f92:Int, f93:Int, f94:Int, f95:Int, f96:Int, f97:Int, f98:Int, f99:Int, f100:Int,
+    f101:Int, f102:Int, f103:Int, f104:Int, f105:Int, f106:Int, f107:Int, f108:Int, f109:Int, f110:Int,
+    f111:Int, f112:Int, f113:Int, f114:Int, f115:Int, f116:Int, f117:Int, f118:Int, f119:Int, f120:Int,
+    f121:Int, f122:Int, f123:Int, f124:Int, f125:Int, f126:Int, f127:Int, f128:Int, f129:Int, f130:Int,
+    f131:Int, f132:Int, f133:Int, f134:Int, f135:Int, f136:Int, f137:Int, f138:Int, f139:Int, f140:Int,
+    f141:Int, f142:Int, f143:Int, f144:Int, f145:Int, f146:Int, f147:Int, f148:Int, f149:Int, f150:Int,
+    f151:Int, f152:Int, f153:Int, f154:Int, f155:Int, f156:Int, f157:Int, f158:Int, f159:Int, f160:Int,
+    f161:Int, f162:Int, f163:Int, f164:Int, f165:Int, f166:Int, f167:Int, f168:Int, f169:Int, f170:Int,
+    f171:Int, f172:Int, f173:Int, f174:Int, f175:Int, f176:Int, f177:Int, f178:Int, f179:Int, f180:Int,
+    f181:Int, f182:Int, f183:Int, f184:Int, f185:Int, f186:Int, f187:Int, f188:Int, f189:Int, f190:Int,
+    f191:Int, f192:Int, f193:Int, f194:Int, f195:Int, f196:Int, f197:Int, f198:Int, f199:Int, f200:Int,
+    f201:Int, f202:Int, f203:Int, f204:Int, f205:Int, f206:Int, f207:Int, f208:Int, f209:Int, f210:Int,
+    f211:Int, f212:Int, f213:Int, f214:Int, f215:Int, f216:Int, f217:Int, f218:Int, f219:Int, f220:Int,
+    f221:Int, f222:Int, f223:Int, f224:Int, f225:Int, f226:Int, f227:Int, f228:Int, f229:Int, f230:Int,
+    f231:Int, f232:Int, f233:Int, f234:Int, f235:Int, f236:Int, f237:Int, f238:Int, f239:Int, f240:Int,
+    f241:Int, f242:Int, f243:Int, f244:Int, f245:Int, f246:Int, f247:Int, f248:Int, f249:Int, f250:Int,
+    f251:Int, f252:Int, f253:Int, f254:Int
+  )
+
+  object MaxArityRecord {
+    // unable to generate code for case classes with more than 120 int fields due to following error:
```

Comment:
(suggestion): Can we place this comment above the `BigRecord` case class definition instead? 
In this case it would make sense to remove `MaxArityRecord` because it simply pollutes the codebase since we don't use it anywhere.